### DISCUSSION TITLE
CES predict_in_sample

### DISCUSSION THREAD

**Discussion by jojocie** [Nixtla/statsforecast#285] [Q&A] [answered]
Hello,

Thank you for all the useful work you have already done !

I am wondering if you have scheduled to implement predict_in_sample function method in AutoCES class?

Thank you in advance for your answer 😄

**Comment by AzulGarza** [comment, marked as answer]
Hey @jojocie! This is possible with the latest version of statsforecast. :)

**Reply by jojocie** [reply]
Hello, thank you very much :)
